meshpy/simulation_manager/simulation_manager.py

```diff
@@ -327,21 +327,21 @@ def add(self, other):
     def create_run_scripts(
         self,
         *,
-        baci_build_dir=None,
+        four_c_build_dir=None,
         status=False,
         status_file=False,
         script_name="run.sh",
-        **kwargs
+        **kwargs,
     ):
         """
         Create the script to run all simulations contained in this manager.
 
         Args
         ----
-        baci_build_dir: str
-            Build directory for baci.
+        FOUR_C_BUILD_DIR: str
```

Comment:
```suggestion
        four_c_build_dir: str
```

Reply:
I prefer to have all function arguments in lower case.

Reply:
Oh, I missed that one, I have a fixup pull request comming up.